client/src/legacy/CMSMain.Tree.js

```diff
@@ -140,6 +141,45 @@ $.entwine('ss.tree', function($) {
         }
       };
       return config;
+    },
+
+    /**
+     * Validates the moving of a node
+     * @param {Object} data provided by the move_node.jstree event
+     * @returns {Promise<boolean>} Returning false will prevent the node from moving
+     */
+    canMove: async function(data) {
+      // only display warning if its the homepage
+      var isHomePage = data.rslt.o.find(".homepage").first().length > 0;
+      if (!isHomePage) {
+        return true;
+      }
+
+      // only display warning if we're moving to a new parent
+      var oldParentId = data.rslt.op.data('id');
+      var newParentId = data.rslt.np.data('id');
+      if (oldParentId === newParentId) {
+        return true;
+      }
+
+      var message = i18n._t(
+        'CMS.RemoveHomePageWarningMessage',
+        'Warning: This page has the URL segment of \'home\'. ' +
```

Comment:
Ok, so `home` is configurable via `RootURLController::get_homepage_link()` - for that reason we'll need to get this from PHP somehow and pass it as an argument to `i18n.inject()`.

Without adding irrelevant schema data to something in the page for this, or adding an XHR request just to check whether it's the home page, I'd probably suggest removing this part of the text as well so it just says "This page is the home page. Are you sure you want to delete it?" - I realise you've been through copy changes a bunch of times already so sorry 😆  but we can't hard code it in one place and make it dynamic in another